src/napari_epitools/_widget.py

```diff
@@ -369,11 +384,97 @@ def _create_cell_statistics_widgets() -> list[Widget]:
     return [image, labels, run, export]
 
 
+def _create_colour_labels_widgets() -> list[Widget]:
+    """Create widgets for colouring Labels layers by their featuers"""
+
+    create_colour_labels_heading = magicgui.widgets.create_widget(
+        label="<b>Colour by statistic</b>",  # bold label
+        widget_type="Label",
+        gui_only=True,
+    )
+
+    labels_tooltip = "Select a 'Labels' layer to set colourmap for"
+    labels = magicgui.widgets.create_widget(
+        annotation=napari.layers.Labels,
+        name="colourmap_input_labels",
+        label="labels",
+        options={"tooltip": labels_tooltip},
+    )
+
+    statisitc_tooltip = "Select which statistic to use for colouring the 'Labels'"
+    statistic = magicgui.widgets.create_widget(
+        name="colourmap_statistic",
+        value="None",
+        label="statistic",
+        widget_type="ComboBox",
+        options={
+            "choices": ["None", "area", "perimeter", "orientation", "neighbours"],
```

Comment:
I feel these should all be capitalised or none of them

Reply:
yeah good point - I'll change "None" to "id" as that's actually what it does